examples/contactnets_simple.py

```diff
@@ -110,7 +113,12 @@ def main(system: str = CUBE_SYSTEM,
     data_asset = DATA_ASSETS[system]
     storage_name = os.path.join(os.path.dirname(__file__), 'storage',
                                 data_asset)
+
```

Comment:
Re the above comment:  in favor of the definitions of `DATA_STORAGE_NAME` and `RESULTS_STORAGE_NAME`, looks like you can just remove lines 114 through 118, which re-implement those definitions, and use those constants instead.

Reply:
I think we need these lines -- unlike the static names, these dynamically pick the data asset to either be the cube or the elbow.

Reply:
Ah, yes good catch.  Agreed.